code/server/validations/authValidation.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// Import the module for validation checks
+const { check } = require('express-validator');
+
+// Validation checks for registration of a new user
+exports.signup = [
+    check('firstname').isString().notEmpty(),
+    check('lastname').isString().notEmpty(),
+    check('email').isEmail(),
+    check('mobile').isString().notEmpty(),
+    check('role').isString().notEmpty(),
+    check('password').isStrongPassword({
+        minLength: 8,
+        minLowercase: 1,
+        minUppercase: 1,
+        minSymbols: 1,
+        minNumbers: 1
+    }),
+];
+
+// Validation checks for login of a user
+exports.login = [
+    check('username').isEmail(),
+    check('password').isStrongPassword({
+        minLength: 8,
```

Comment:
Remove these constraints for the password in the login, just ask for the it to be a not empty string